vars/sonarExecuteScan.groovy

```diff
@@ -191,3 +206,30 @@ private void loadSonarScanner(config){
         mv ${foldername} .sonar-scanner
     """
 }
+
+private void loadCertificates(Map config) {
+    String certificateFolder = '.certificates/'
+    List wgetOptions = [
+        "--directory-prefix ${certificateFolder}"
+    ]
+    List keytoolOptions = [
+        '-import',
+        '-noprompt',
+        '-storepass changeit',
+        '-keystore .sonar-scanner/jre/lib/security/cacerts'
+    ]
+    if (config.customTlsCertificateLinks){
+        if(config.verbose){
+            wgetOptions.push('--verbose')
+            keytoolOptions.push('-v')
+        }else{
+            wgetOptions.push('--no-verbose')
+        }
+        config.customTlsCertificateLinks.each { url ->
+            def filename = new File(url).getName()
+            filename = URLDecoder.decode(filename, java.nio.charset.StandardCharsets.UTF_8.name())
```

Comment:
Why fullQualified class here instead of import?

Reply:
No reason, I'll change this.